### DISCUSSION TITLE
Front-End Framework

### DISCUSSION THREAD

**Discussion by JosephSuvak** [JosephSuvak/QR-Ticket-Generator#10] [General]
For discussing Front-End changes.

**Comment by JohnBanas** [comment]
### Handlebars

## Changes 

- The Homepage is almost finished it needs a link that sends you back to your account if you are logged in.
- The user account page pulls up when you create a new user or log in. If you login and have tickets to concerts it shows them to you. If you do not it notifies you there are no tickets currently.
- You can select to look at all the available shows from user account page
- You can logout from user account page
- You can log out from the available concerts page.
- You can not purchase a ticket or get a qr code yet.

**Comment by JohnBanas** [comment]
## Front End Changes


Front end still needs a lot of work, definitely from a style perspective, but it will pass for a MVP. 

- The handlebar files will now create a landing page, a user account page which shows all tickets the current logged in user has. 

- Then a page for all the available concerts, which you can choose one to view in a single concert page. 

- Then select that concert to add a ticket to your account page. 

- You can select a single ticket from your account page and choose to view that ticket which takes you to the single ticket page, where you can choose to delete the ticket which works. Or you can choose to make a QR code for the ticket, this does not work yet, that is the next step.
